katello/hooks/pre/10-reset_feature.rb

```diff
@@ -135,6 +136,11 @@ def reset_pulp
   )
 end
 
+def reset_pulcore
+  Kafo::KafoConfigure.logger.info 'Dropping Pulpcore database!'
+  execute('sudo runuser - postgres -c "dropdb pulpcore"')
```

Comment:
Will this error if there is no pulpcore installed? We should check to be safe. There is code in (https://github.com/theforeman/foreman-installer/blob/develop/hooks/boot/01-kafo-hook-extensions.rb#L40) at the end that could be turned into a helper method.

Reply:
Why does it use `sudo` and `runuser`? Just using `runuser` should be sufficient, right?

Reply:
ehelms : For some reason, checking for param_value('foreman_proxy::plugin::pulp', 'pulpcore_enabled') returns nil inside this .. I checked the katello-answers.yaml in my setup and the pulpcore_enabled is set to True..

Reply:
So @jlsherrill helped me figure that param_value('foreman_proxy_plugin_pulp', 'pulpcore_enabled') works..Updated with the same..